### ISSUE TITLE
Error when attempting to download little file

### ISSUE THREAD

**Issue by YggNode** [FlareSolverr/FlareSolverr#110]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

### How to enable debug and html traces

[Follow the instructions from this wiki page](https://github.com/FlareSolverr/FlareSolverr/wiki/How-to-enable-debug-and-html-trace)

### Environment

* **FlareSolverr version**: 1.2.5
* **Last working FlareSolverr version**: no tests done
* **Operating system**: Debian
* **Are you using Docker**: [yes/no] yes
* **Are you using a proxy or VPN?** [yes/no] no
* **Are you using Captcha Solver:** [yes/no] no (no need of one)

### Description

usual call to flaresolverr server such as 

```
import requests
import json

url = "http://192.168.1.X:8191/v1" #take care to replace with your server's ip adress

payload = json.dumps({
  "cmd": "request.get",
  "url": "https://www4.yggtorrent.li/rss/download?id=742738&passkey=s5d8r5t4g5f6d4f5g4t7r8e9d6f5gf2f",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)",
  "maxTimeout": 60000,
  "session": "394171b0-9991-11eb-88f1-dbfce850809a",
  "download": True
})
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

```

Don't worry passkey is valid but not registered so you can easily use it for downloading file but not to use this file

### Logged Error Messages

Unable to process browser request. Error: Error: net::ERR_ABORTED at https://www4.yggtorrent.li/rss/download?id=742738&passkey=s5d8r5t4g5f6d4f5g4t7r8e9d6f5gf2f"

### Screenshots

![Postman_b7wRe2v0Rx](https://user-images.githubusercontent.com/81962980/114277887-6eea8080-9a2d-11eb-9478-4dc5dd910b2e.png)

**Comment by YggNode** [comment]
or maybe file size is limited under i don't know maybe 20 kB ?

**Comment by ngosang** [comment]
First of all, I think you should hide your passkey.
I have to review the "download file" feature because there are more related issues.

By now you can do this (is what Jackett does):
1) Make a regular request to the main page with FlareSolverr => https://www4.yggtorrent.li/engine/search?do=search
2) Get the cookies from the response.
3) Make the download directly from Python. You have to copy the cookies from FlareSolverr and add them to the Python request..

The main idea is you only have to call FlareSolverr once to get the cookies. Then you can use those cookies to make all the requests you want from Python.

**Comment by YggNode** [comment]
Yep, sorry for being late, but i successfully did what you said and it's working right now.

passkey is valid only to download torrent file, but not content so, don't worry.
It's just a 32 characters string randomly generated by my password manager.

**Comment by neilee** [comment]
> You have to copy the cookies from FlareSolverr and add them to the Python request..

@ngosang can I please ask how you achieve this? I can see in the flaresolverr response that the cookies are stored in response['solution']['cookies'] but I'm not too sure how to pass these into a requests.get() ?

Thank you!

**Comment by YggNode** [comment]
Hello,

Sorry : i didn't see your answer before today.
For your question, i think you could find what you want in my scipt. I was like you're actually and i found by testing possibilities : 
https://github.com/YggNode/yggnode/blob/main/yggnode.py

Hope it will be usefull.

**Comment by neilee** [comment]
Thanks @YggNode - I see what you're doing in your code, much appreciated!

**Comment by ngosang** [comment]
Check if it's fixed in version 1.2.9

**Comment by ngosang** [comment]
[FlareSolverr v2.0.0](https://github.com/FlareSolverr/FlareSolverr/releases/tag/v2.0.0) is released! If you are using Jackett, version v0.18.1041 or higher is required to bypass Cloudflare.

Sadly downloading small files feature has been removed from FlareSolverr v2 because it's not supported by Puppeter for Firefox.
